lib/validate.js

```diff
@@ -1223,7 +1223,4 @@ function validate(openapi, options, callback) {
     });
 }
 
-module.exports = {
-    validateSync: validateSync,
-    validate: validate
-}
+module.exports = { validateSync, validate }
```

Comment:
Why is this a one-liner instead of multiline like other files?

Reply:
I was gonna make them all one line, musta missed. It's short enough to be a one liner right?